.github/workflows/update-api-client.yml

```diff
@@ -0,0 +1,93 @@
+name: Automation - Update API Client
+
+on:
+  push:
+    branches:
+      - main
+    path:
+      - backend/src/**.ts
+  workflow_dispatch:
+
+jobs:
+  update-api-client:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          ref: main
+          submodules: true
+          ssh-key: ${{ secrets.JIMMI_API_CLIENT_REPO_DEPLOY_KEY }}
+
+      - name: Setup deno
+        uses: denoland/setup-deno@v1
+        with:
+          deno-version: vx.x.x
+
+      - name: Start backend
+        working-directory: backend
+        run: |
+          deno run --no-check --allow-read --allow-net --config deno.json src/app.ts &
+          BACKEND_PID=$!
+          sleep 1
+          # Check if process is still running and fail if not
+          ps | grep " $BACKEND_PID "
+
+      - name: Delete old api client files
+        working-directory: frontend/lib/jimmi-api-client
+        run: git checkout main && rm -rf src/*
+
+      - name: Generate api client
+        working-directory: frontend/lib/jimmi-api-client
+        run: |
+          docker run --rm \
+          --add-host host.docker.internal:host-gateway \
+          -v "$(pwd)":/local \
+          openapitools/openapi-generator-cli generate \
+          -i http://host.docker.internal:8000/swagger.json \
+          -c /local/config.yml \
+          -g typescript \
+          -o /local/src/
+
+      - name: Commit and push changes in submodule
+        working-directory: frontend/lib/jimmi-api-client
+        run: |
+          sudo mv src/index.ts src/mod.ts
```

Comment:
```suggestion
          mv src/index.ts src/mod.ts
```
Why sudo?

Reply:
Sadly, this does not work, it has to be `sudo`  :-/